recipes-devtools/ruby/ruby_3.1.0.bb

```diff
@@ -0,0 +1,108 @@
+require ruby.inc
+
+DEPENDS:append:libc-musl = " libucontext"
+
+SRC_URI += " \
```

Comment:
potential issue: Use SRC_URI_append otherwise this will override weak defaults by inherit